config.js

```diff
@@ -13,7 +13,7 @@ const config = {
     },
     live_preview: {
         enable: false,
-        host: 'rest-preview.contentstack.com'
+        host: 'api.contentstack.io'
```

Comment:
@abhinav-from-contentstack is this default host?

Reply:
that's under the live_preview object, which is as it was in master branch.